src/common/charset/Charset.h

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef COMMON_CHARSET_CHARSET_H_
+#define COMMON_CHARSET_CHARSET_H_
+
+#include "base/Base.h"
+#include "base/StatusOr.h"
+
+namespace nebula {
+
+struct CharsetToCollation {
+    std::string               charsetName_;
+    std::string               defaultColl_;
+    std::vector<std::string>  supportColl_;
+    std::string               desc_;
+    int32_t                   maxLen_;
+};
+
+
+class CharsetInfo final {
+public:
+    static CharsetInfo* instance() {
+        static std::unique_ptr<CharsetInfo> charsetInfo(new CharsetInfo());
+        charsetInfo->prepare();
+        return charsetInfo.get();
+    }
+
+    void prepare() {
+        charsetToCollation["utf8"] = {"utf8", "utf8_bin", {"utf8_bin"}, "UTF-8 Unicode", 4};
```

Comment:
Do you really need the prepare method?  Move it into ctor is enough.

Reply:
Yeah，I have  formatted the code.
now is ok